*** controller-config.xml	2011-05-09 14:03:38.000000000 -0400
--- controller-config.xml.ef	2011-05-15 15:22:00.000000000 -0400
***************
*** 42,47 ****
--- 42,55 ----
    <root pattern="/apps/tamboti" path="xmldb:exist:///db/library"/>
  
    <root pattern="/apps" path="xmldb:exist:///db"/>
+      
+   <!-- EDF: change so that all requests default to the db, unless preceded by fs,
+ 	allow legacy (demo) to access through /exist -->
+  	<!-- -->
+  	<root pattern="/exist/rest" path="xmldb:exist:///"/>
+   <root pattern="/exist/" path="/"/>
+   <root pattern="/db" path="xmldb:exist:///db"/>
+ 	<root pattern="/.*" path="xmldb:exist:///db"/>
    
    <!-- 
      ++ To redirect requests based on the server name (i.e. www.example.com)
