--- ../db/code/modules/resources/xspec/src/compiler/generate-xspec-tests.xsl	2011-09-05 15:42:49.223090164 -0400
+++ ../db/code/modules/resources/xspec/src/compiler/generate-xspec-tests.xsl	2011-09-05 17:24:55.518909152 -0400
@@ -5,6 +5,7 @@
 <!--  URI:        http://xspec.googlecode.com/                             -->
 <!--  Tags:                                                                -->
 <!--    Copyright (c) 2008, 2010 Jeni Tennsion (see end of file.)          -->
+<!--    Modifications (c) 2010 Efraim Feinstein (same license)             -->
 <!-- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -->
 
 
@@ -50,6 +51,8 @@
     <xsl:apply-templates select="." mode="x:copy-namespaces" />
   	<import href="{$stylesheet-uri}" />
   	<import href="{resolve-uri('generate-tests-utils.xsl', static-base-uri())}"/>
+
+    <strip-space elements="*"/>
     <!-- This namespace alias is used for when the testing process needs to test
          the generation of XSLT! -->
     <namespace-alias stylesheet-prefix="__x" result-prefix="xsl" />
