--- ../db/code/modules/resources/xspec/src/compiler/generate-common-tests.xsl	2011-09-05 15:42:49.223090164 -0400
+++ ../db/code/modules/resources/xspec/src/compiler/generate-common-tests.xsl	2011-09-05 17:17:34.638922173 -0400
@@ -5,6 +5,7 @@
 <!--  URI:        http://xspec.googlecode.com/                             -->
 <!--  Tags:                                                                -->
 <!--    Copyright (c) 2008, 2010 Jeni Tennsion (see end of file.)          -->
+<!--    Modifications 2010 Efraim Feinstein                                -->
 <!-- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -->
 
 
@@ -18,6 +19,7 @@
 
    <pkg:import-uri>http://www.jenitennison.com/xslt/xspec/generate-common-tests.xsl</pkg:import-uri>
 
+   <xsl:strip-space elements="*"/>
    <xsl:preserve-space elements="x:space"/>
 
    <!--
@@ -44,11 +46,12 @@
       <xsl:apply-templates select="$unshared-scenarios/*" mode="x:generate-tests"/>
    </xsl:template>
 
-   <xsl:template match="x:description" mode="x:copy-namespaces">
+   <xsl:template match="element()" mode="x:copy-namespaces">
       <xsl:variable name="e" as="element()" select="."/>
       <xsl:for-each select="in-scope-prefixes($e)">
          <xsl:namespace name="{ . }" select="namespace-uri-for-prefix(., $e)"/>
       </xsl:for-each>
+      <xsl:apply-templates select="element()" mode="#current"/>
    </xsl:template>
 
    <xsl:function name="x:gather-specs" as="element(x:description)+">
